Resources/config/commands.yml

```diff
@@ -10,6 +10,13 @@ services:
         tags:
             - { name: console.command }
 
+    apisearch.print_indices_command:
+        class: Apisearch\Command\PrintIndicesCommand
+        arguments:
+        - "@apisearch.repository_bucket"
```

Comment:
Fix indentations, should looks like in other registers.

Reply:
:+1: